esp-hal/CHANGELOG.md

```diff
@@ -54,6 +56,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed `AnyInputOnlyPin` in favour of `AnyPin`. (#2071)
 - Removed the following functions from `GpioPin`: `is_high`, `is_low`, `set_high`, `set_low`, `set_state`, `is_set_high`, `is_set_low`, `toggle`. (#2094)
 - Removed `Rtc::get_time_raw` (#1883)
+- Removed `AnyPin` (#2128)
```

Comment:
I'm not sure how to word this but it's somewhat misleading. Might be less confusing to just skip this entry entirely and possibly reword the `Changed` entry to include this "Removed `AnyPin`" entry.

Reply:
Yeah, you are right, logging "replaced A with B but renamed B to A" kinds of things is not trivial.